src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,166 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.text.SimpleDateFormat;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileEditor {
+
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+
+    private static final HashMap<String, String> COMMAND_INFO = new HashMap<>();
+
+    static {
+        COMMAND_INFO.put("create", "creates a text file at the specified path");
+        COMMAND_INFO.put("read", "reads a text file at the specified path and "
+                + "+ prints the content to the console");
+        COMMAND_INFO.put("info", "prints the information about the file: \"\n"
+                + "+ \"number of chars, words, lines, date and time of last modifying,"
+                + " size of a file");
+        COMMAND_INFO.put("help", "prints information about all the commands");
+        COMMAND_INFO.put("helpCommand", "prints information about the specified command");
+        COMMAND_INFO.put("exit", "exit from the program");
+    }
+
+    public void consoleWork() throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.println("Enter a path and a name of a file you will work with: ");
+            String path = scanner.next();
+            String filename = scanner.next();
+            menu();
+            String choice = scanner.next();
+            switch (choice) {
+                case "create" : {
+                    create(path, filename);
+                    break;
+                }
+                case "read" : {
+                    System.out.println(read(path, filename));
+                    break;
+                }
+                case "info" : {
+                    info(path, filename);
+                    break;
+                }
+                case "help" : {
+                    help();
+                    break;
+                }
+                case "helpCommands" : {
+                    menu();
+                    String command = scanner.next();
+                    help(command);
+                    break;
+                }
+                case "exit" : {
+                    System.exit(0);
+                    break;
+                }
+                default:
+                    System.out.println("Would you like to save the printed text in the file? "
+                            + " Enter 'yes' if your answer is positive");
+                    String answer = scanner.next();
+                    saveInput(path, filename, choice, answer);
+            }
+        }
+    }
+
+    private static void saveInput(String path, String filename, String choice, String answer) {
```

Comment:
```suggestion
    private void saveInput(String path, String filename, String choice, String answer) {
```
чи можемо ми уникнути статичних методів у цьому завданні?